client/app/data/l10n/en.json

```diff
@@ -594,5 +594,6 @@
   "This email is to remind you the presence of unread or updated reports.": "This email is to remind you the presence of unread or updated reports.",
   "Reminder about unread or updated reports": "Reminder about unread or updated reports",
   "Role: {Role}": "Role: {Role}",
-  "Password: {Password}": "Password: {Password}"
+  "Password: {Password}": "Password: {Password}",
+  "Custom support URL": "Custom support URL"
```

Comment:
this manual change to this file should not be included. after the pull request we will use some grunt commands to automatically generate these files.

grunt makeTranslationsSource will update the en.po file
grunt updateTranslations will update the *.json files